CHANGELOG.md

```diff
@@ -38,6 +38,7 @@
 * [CHANGE] Dashboards: "Slow Queries" dashboard no longer works with versions older than Grafana 9.0. #2223
 * [ENHANCEMENT] Dashboards: added missed rule evaluations to the "Evaluations per second" panel in the "Mimir / Ruler" dashboard. #2314
 * [ENHANCEMENT] Dashboards: add k8s resource requests to CPU and memory panels. #2346
+* [ENHANCEMENT] Dashboards: add RSS memory utilization panel for ingesters, store-gateways and compactors. #2479
```

Comment:
are you planning to do the same for the other components? Or is that not necessary because they don't mmap?

Reply:
No, because it doesn't look necessary. See details here:
https://github.com/grafana/mimir/issues/2466#issuecomment-1190024910